src/AcceptanceTests/StepDefinitions/HostingStepDefinitions.cs

```diff
@@ -0,0 +1,37 @@
+using System.Diagnostics;
+
+namespace SFA.DAS.Funding.ApprenticeshipEarnings.AcceptanceTests.StepDefinitions;
+
+[Binding]
+public class HostingStepDefinitions
+{
+    private readonly TestContext _testContext;
+    private readonly FeatureContext _featureContext;
+
+    public HostingStepDefinitions(TestContext testContext, FeatureContext featureContext)
+    {
+        _testContext = testContext;
+        _featureContext = featureContext;
+    }
+
+    [BeforeScenario]
+    public async Task CreateConfig()
+    {
+        var stopwatch = new Stopwatch();
+        stopwatch.Start();
+        _testContext.TestFunction = new TestFunction(_testContext, $"TEST{_featureContext.FeatureInfo.Title.Replace(" ", "")}");
+        await _testContext.TestFunction.StartHost();
+        stopwatch.Stop();
+        Console.WriteLine($"Time it took to spin up Azure Functions Host: {stopwatch.Elapsed.Milliseconds} milliseconds for hub {_testContext.TestFunction.HubName}");
+    }
+
+    [AfterScenario]
+    public async Task CleanupAfterTestHarness()
+    {
+        var stopwatch = new Stopwatch();
+        stopwatch.Start();
+        await _testContext.TestFunction.DisposeAsync();
```

Comment:
this still needed?

Reply:
yes, why do you suggest it might not be?